WooCommerce/Classes/ViewRelated/Orders/Payment Methods/PaymentMethodsViewModel.swift

```diff
@@ -205,14 +205,24 @@ final class PaymentMethodsViewModel: ObservableObject {
             DDLogError("⛔️ Order not found, can't collect payment.")
             return presentNoticeSubject.send(.error(Localization.genericCollectError))
         }
+        let alertsPresenter = CardPresentPaymentAlertsPresenter(rootViewController: rootViewController)
+        let configuration = CardPresentConfigurationLoader().configuration
 
         collectPaymentsUseCase = useCase ?? CollectOrderPaymentUseCase(
             siteID: self.siteID,
             order: order,
             formattedAmount: self.formattedTotal,
             rootViewController: rootViewController,
             onboardingPresenter: self.cardPresentPaymentsOnboardingPresenter,
-            configuration: CardPresentConfigurationLoader().configuration)
+            configuration: CardPresentConfigurationLoader().configuration,
```

Comment:
super nit: could `configuration` be used for this parameter? I understand that it might be better keeping minimal changes, just thought that the same configuration is being used for `CardPresentPaymentPreflightController` within `CollectOrderPaymentUseCase` before the PR as well. 

```suggestion
            configuration: configuration,
```